scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -561,6 +564,41 @@ static u64 sched_prio_to_latency_weight(u64 prio)
 	return sched_prio_to_weight[DL_MAX_LAT_PRIO - prio - 1];
 }
 
+static void do_iowait_boost(struct task_struct *p, struct task_ctx *taskc, s32 cpu)
```

Comment:
nit: Just because we're only sometimes boosting, could we please rename this to `maybe_boost_iowait()` (or something to that effect).